rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/client/analysis/GetFileExclusionsResponse.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SonarLint Core - RPC Protocol
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.rpc.protocol.client.analysis;
+
+import java.util.Set;
+import javax.annotation.Nullable;
+
+public class GetFileExclusionsResponse {
+  private final Set<String> fileExclusionPatterns;
+
+  public GetFileExclusionsResponse(@Nullable Set<String> fileExclusionPatterns) {
```

Comment:
Why nullable? Shouldn't it be an empty set then? And if it's nullable, the getter should be marked as @CheckForNull as well.

Reply:
Exactly! It's a copy-paste issue